code/modules/food_and_drinks/kitchen_machinery/microwave.dm

```diff
@@ -134,6 +137,18 @@
 			return TRUE
 		return
 
+	if(istype(O, /obj/item/card/emag))
+		if(obj_flags & EMAGGED)
+			to_chat(user, "<span class='warning'>You've already sabotaged the [src].'</span>")
+		else
+			if(prob(1))
+				to_chat(user, "<span class='notice'>You emag the fucking microwave</span>")
+			else
+				to_chat(user, "<span class='notice'>*bzzt*</span>")
+			
+			obj_flags |= EMAGGED
+	return TRUE
```

Comment:
Incorrect indentation.

Reply:
Always failing the build, thanks for the help btw